### ISSUE TITLE
Choice behind RoBERTa and BERT

### ISSUE THREAD

**Issue by MartaM21** [marcotcr/checklist#48]
Just a curiosity. 
Is there a specific implementation choice on why the the fill-in for the templates ({mask}) produced by CheckList are created from suggestions of RoBERTa and BERT? Given that these same two research models are analysed and their shortcomings are highlighted in the paper (why in some way rely on """imperfect""" models?)

Thanks a lot

**Comment by BBloggsbott** [comment]
You can also use your own model. Checklist uses [`transformers`](https://github.com/huggingface/transformers). Just pass the [model name](https://huggingface.co/transformers/pretrained_models.html) and it will load that model. You can also finetune a model and use that (I tried this and it worked).

**Comment by marcotcr** [comment]
RoBERTa and BERT actually work pretty well for masked language modeling (which is what we use them for). The shortcomings highlighted in the paper are for _finetuned_ models performing other tasks (sentiment analysis, QQP, SQuAD).
Further, these suggestions are vetted by humans - so RoBERTa and BERT don't have to be great, they just have to be good enough to complement the human in coming up with tests. And they certainly are.
